garage/logger/snapshotter.py

```diff
@@ -0,0 +1,88 @@
+"""Allows the taking of snapshots."""
+from os import path as osp
+import pickle
+
+
+class Snapshotter(object):
```

Comment:
No need to inherit from object in Python 3

Reply:
Fixed